src/app/lib/utilities/preprocessor/rules/cpsAssetPage/cpsOnlyOnwardJourneys/index.test.js

```diff
@@ -0,0 +1,324 @@
+import path from 'ramda/src/path';
+import cpsOnlyOnwardJourneys from '.';
+import pglNoCpsOnwardJourneys from '#data/arabic/cpsAssets/art-and-culture-38260491.json';
+import styNoCpsOnwardJourneys from '#data/igbo/cpsAssets/afirika-23252735.json';
+import pglSomeCpsOnwardJourneys from '#data/azeri/cpsAssets/azerbaijan-44208474.json';
+import pglNoOnwardJourneys from '#data/pidgin/cpsAssets/sport-23252855.json';
+import pglAllCpsOnwardJourneys from '#data/pidgin/cpsAssets/tori-49221071.json';
+import mapPageAllCpsOnwardJourneys from '#data/pidgin/cpsAssets/23248703.json';
+
+describe('cpsOnlyOnwardJourneys', () => {
+  const expectedSeeAlsos = {
```

Comment:
Would it be better to have this fixture data live in a separate file?

Reply:
Yeah, I'll take a look at that now

Reply:
I've extracted out some fixture files, I hope that makes it cleaner: 6540e69e3647ecdb12447140ac4db9e46dd59eb4 

I've put the full asset next to the filtered 'see-also' group so it's easier to understand what the partial fixture relates to, not sure if that's a good idea or not?

Reply:
Make sense👍